websites/Y/Yandex Music/presence.ts

```diff
@@ -37,7 +37,7 @@ setInterval(async () => {
 	}
 
 	presenceData = {
-		largeImageKey: "og-image",
+		largeImageKey: (<HTMLImageElement>document.querySelector("body > div.page-root.page-root_no-player.deco-pane-back.theme.theme_dark.black > div.bar > div.bar__content > div.player-controls.deco-player-controls > div.player-controls__track-container > div > div > div.track.track_type_player > div.entity-cover.deco-entity-image-placeholder.track-cover.entity-cover_size_Large > a > img")).src.replace("50x50","100x100"),
```

Comment:
Prefer an Imgur URL instead of this very long query selector, which can change frequently.

Reply:
Pretty sure they wants to use cover art..
You can use `navigator.mediaSession.metadata.artwork.at(-1).src` instead.

Reply:
Pretty sure they wants to use cover art..
You can use `navigator.mediaSession.metadata.artwork.at(-1).src` instead.